README.md

```diff
@@ -54,6 +54,12 @@ yarn migrate:worker
 yarn knex migrate:latest
 ```
 
+Run codegen
```

Comment:
Should have a colon.

Reply:
shoutout to squash merging or we'd have to call them "colonsd" forever